examples/example-app-v13/setup-jest.ts

```diff
@@ -1,2 +1,14 @@
-import 'jest-preset-angular/setup-jest';
+// Can't use standard setup-jest, b/c it uses require()
+// import 'jest-preset-angular/setup-jest';
+
+import 'zone.js/fesm2015/zone-testing-bundle.min.js';
 import './jest-global-mocks';
+
+import { getTestBed } from '@angular/core/testing';
+import { BrowserDynamicTestingModule, platformBrowserDynamicTesting } from '@angular/platform-browser-dynamic/testing';
+
+getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting(), {
+  teardown: {
+    destroyAfterEach: true,
```

Comment:
this one is default set in Angular 13, we don't need to set here

Reply:
Are you sure about this? My understanding is that this is set in new projects (in the karma setup file), and is recommended, but it's not a default. I'll step through to figure that out - and if you're right, I'll also remove from docs.

Reply:
Ye it's a part of Angular 13 change, you can see it here https://github.com/ngrx/platform/issues/3243#issuecomment-973038378

Reply:
You're right; I'll remove.